Report/Chapters/Introduction.tex

```diff
@@ -2,7 +2,6 @@ \chapter{Introduction}
 
 Programming has always been an activity which is done on a computer with a keyboard. Despite the existence of different visual programming languages, like Blender, there are no programs in which well known programming languages like Java, Haskell or C(++) can be visually implemented. 
```

Comment:
You could see Blender as a programming language that is purely visual.

I think that the claim is right, but I am not 100% sure. But I like the 'in your face'-ness of it. I think I want to keep it this way.